src/txmempool.h

```diff
@@ -681,6 +681,29 @@ class CTxMemPool
      */
     bool CalculateMemPoolAncestors(const CTxMemPoolEntry& entry, setEntries& setAncestors, uint64_t limitAncestorCount, uint64_t limitAncestorSize, uint64_t limitDescendantCount, uint64_t limitDescendantSize, std::string& errString, bool fSearchForParents = true) const EXCLUSIVE_LOCKS_REQUIRED(cs);
 
+    /** Try to calculate all in-mempool ancestors of a set of entries, and check for ancestor and
+     * descendant limits (all are inclusive of the transactions in entries). The same limits are
+     * used no matter how many transactions are passed in. For example, if entries.size() = 3 and
+     * the limit is 25, the union of all 3 sets of ancestors must be <= 22.
+     * @param[in,out]   setAncestors            Set of in-mempool ancestors. Updated to include
+     *                                          any new ancestors found.
+     * @param[in]       limitAncestorCount      Max number of txns including ancestors.
+     * @param[in]       limitAncestorSize       Max virtual size including ancestors.
+     * @param[in]       limitDescendantCount    Max number of txns including descendants.
+     * @param[in]       limitDescendantSize     Max virtual size including descendants.
+     * @param[out]      errString               Populated with error reason if a limit is hit.
+     * @param[in]       fSearchForParents       Whether to search for entries' in-mempool parents.
+     *                                          Must be true if any entries are not already in mempool.
+     */
+    bool CalculateMemPoolAncestors(const std::vector<CTxMemPoolEntry::CTxMemPoolEntryRef>& entries,
```

Comment:
Added doxygen comment for `entries` param, but my intention was to overload the `CalculateMemPoolAncestors` function (the fact that the first argument is a vector instead of a single entry should suffice?)